qemu_amd64.go

```diff
@@ -0,0 +1,223 @@
+//
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package virtcontainers
+
+import (
+	"fmt"
+	"os"
+
+	govmmQemu "github.com/intel/govmm/qemu"
+)
+
+type qemuAmd64 struct {
+	// inherit from qemuSystem, overwrite methods if needed
+	qemuSystem
+}
+
+const defaultQemuPath = "/usr/bin/qemu-system-x86_64"
+
+const defaultQemuMachineType = "pc"
+
+const defaultQemuMachineAccelerators = "kvm,kernel_irqchip,nvdimm"
+
+const defaultPCBridgeBus = "pci.0"
+
+const (
+	// QemuPCLite is the QEMU pc-lite machine type
+	QemuPCLite = "pc-lite"
+
+	// QemuPC is the QEMU pc machine type
+	QemuPC = defaultQemuMachineType
+
+	// QemuQ35 is the QEMU Q35 machine type
+	QemuQ35 = "q35"
```

Comment:
Might be dumb question here, but looking at this definition of different machine types, I assume we cannot have the same machine types for other architectures, right ?
Otherwise, we should move this to `qemuSystem`.

Reply:
you are right, sure we can move this to `qemuSystem`